.github/workflows/verify.yml

```diff
@@ -8,9 +8,9 @@ jobs:
     steps:
 
     - name: Set up Go
-      uses: actions/setup-go@v1
+      uses: actions/setup-go@v4
       with:
-        go-version: 1.20
+        go-version: "1.20"
```

Comment:
I'm, fine with this. One alterntive I've seen (in ttrpc) is something like:
```
env:
  GO_VERSION: 1.20.x
...
     - uses: actions/setup-go@v3
        with:
          go-version: ${{ env.GO_VERSION }}
...
```

Reply:
I don't have strong opinions. But I probably prefer to keep things simple, like now, as we only use the go version in one place.

Reply:
One more alternative is to configure setup-go to read from go.mod

Reply:
Hmm, ok,, so it seems, didn't know about that. The only caveat is that the version in go.mod is a different thing than the version we're building with. But in our case (as we don't expect cri-resmgr to be used as a library/module dependency for any other project) we can probably have a practice to keep go.mod in sync with the golang version we use to build our binaries

Reply:
> I don't have strong opinions. But I probably prefer to keep things simple, like now, as we only use the go version in one place.

Sorry, my comment was unclear because the example took the version from the environment. But what I meant was that using 1.20.x is another way to avoid the (lack of) quoting problem. Anyway, I think taking it directly from go.mod is probably even better.